UN-OCHA/unocha-site
# UNO-608 Translation configuration

Translation config for content, paragraphs, media etc

This is in preparation for multilingual content which we are **not doing now**, but I want to ensure the set up is as ready as possible and part of our development workflow when adding new pieces.

### To test:
Check out this branch, import config and clear cache.
Visit `/admin/config/regional/content-language` and see which fields are translatable and which ones are omitted.
Does it look reasonable?
I've followed the advice from https://docs.google.com/document/d/1JMTLyx1dgVMe5Xo85Zn125TX0632y4mDlnUIGqVeAF8/edit#heading=h.5dewinkbvzj9 (see Multilingual heading)

### Optional:
Select an existing page. I used the Myanmar Response because of its varied content and multiple paragraphs. The [Demo page](https://new.dev.unocha-org.ahconu.org/demo-page) would also be a good choice. Create a translation in the language of your choice. Investigate what is able to be translated. Is there anything missing that you'd expect to translate but cannot?

Refs: UNO-608
